build-tools/create-dotnet-pkg/create-dotnet-pkg.csproj

```diff
@@ -34,7 +34,7 @@
       <_FilesToCopy Include="$(DotNetPreviewPath)\sdk\$(DotNetPreviewVersionFull)\EnableWorkloadResolver.sentinel" />
       <_FilesToCopy Include="$(DotNetPreviewPath)\sdk-manifests\$(DotNetPreviewVersionBand)\Microsoft.NET.Workload.Android\**\*" />
       <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Ref\**\*" />
-      <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Sdk\**\*" />
+      <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Sdk.osx-x64\**\*" />
```

Comment:
Does this need to check `$(HostOS)` and use either: `osx-x64`, `windows-x64`, or `linux-x64`?

Reply:
I think this is only used for macOS .pkg creation, though if we wanted to use this to create a .pkg installer for linux in the future we'd need to update this file to check `$(HostOS)` as suggested.

Reply:
I misread that is this the `create-dotnet-pkg` project.